webapp/src/components/ImageFieldEditor.tsx

```diff
@@ -51,7 +51,8 @@ export class ImageFieldEditor<U extends pxt.Asset> extends React.Component<Image
             currentView: "editor",
             headerVisible: true,
             filterOpen: false,
-            gallerySelectedTags: []
+            gallerySelectedTags: [],
+            galleryFilter: " "
```

Comment:
```suggestion
            galleryFilter: ""
```

Just for cleanliness, no need to have it be truthy here I think.

Also probably should update https://github.com/microsoft/pxt/pull/8042/commits/8ea2dd866d21ccf8437814d76d94f0767edf85ad#diff-d27a987847063b48bbd409fa5ea5df1e2d7655def806979aba7492322faa0e36R24 to remove the ?, as it's no longer optional? or leave as optional  and handle in the filter part below like

```
const tags = this.state.galleryFilter?.split(" ")
                .filter(el => !!el)
                .map(el => el.toLowerCase()) ?? [];
```